### PR TITLE
[WIP] scripts: fix gceworker.sh sync

### PR DESCRIPTION
**Pull request by adityamaru**
WIP

Release note: None

### CODE REVIEW

File: scripts/gceworker.sh [hunk 1]
```diff
@@ -32,13 +32,16 @@ case "${cmd}" in
...
            --image-project "ubuntu-os-cloud" \
-           --image-family "ubuntu-1804-lts" \
+           --image-family "ubuntu-2004-lts" \
            --boot-disk-size "100" \
            --boot-disk-type "pd-ssd" \
            --boot-disk-device-name "${NAME}" \
            --scopes "cloud-platform"
     gcloud compute firewall-rules create "${NAME}-mosh" --allow udp:60000-61000
 
+    sudo rm ~/.ssh/config
```
Comment (otan) [inline scripts/gceworker.sh:42]: hmm, this smells like something i'm not sure i want a script to do for me.

Comment (irfansharif) [reply scripts/gceworker.sh:42]: Stop reviewing hacky work around code dear friend

Comment (otan) [reply scripts/gceworker.sh:42]: Hehehe I was gonna try it!!!

**Comment by irfansharif** [comment]
I think we should give up here. There are too many quirks with using unison, and it's frankly easier to push out+pull down working SHAs off of github. Once we have bazel and once that gives us true remote builds+test runs, this will be obsolete anyway.

**Pull request closed without merging**